transaction-history-manager/src/main/resources/application-local.yaml

```diff
@@ -1,4 +1,8 @@
+spring:
+  data:
+    mongodb:
+      database: qpay
 server:
-  port: 8090
+  port: 8088
```

Comment:
You have 5 or 6 microservices right now. The first one is 8081. It cannot be 8088